pkg/alertmanager/alertspb/compat.go

```diff
@@ -27,7 +27,19 @@ func ToProto(cfg string, templates map[string]string, user string) AlertConfigDe
 	}
 }
 
-// ParseTemplates returns a alertmanager config object
+// ToGrafanaProto transforms a Grafana Alertmanager config to a GrafanaAlertConfigDesc.
+func ToGrafanaProto(cfg, user, hash string, id int64, at int64, d bool) GrafanaAlertConfigDesc {
```

Comment:
can you rename `at` and `d` here too, please?

Reply:
Solved in [083abea](https://github.com/grafana/mimir/pull/6682/commits/083abea0147e15612272e7bfab850550f649e3db)